packages/callbacks/tizen/project_def.prop

```diff
@@ -0,0 +1,40 @@
+# See https://docs.tizen.org/application/tizen-studio/native-tools/project-conversion
+# for details.
+
+APPNAME = tizen_interop_callbacks_plugin
+type = staticLib
+profile = common-4.0
+
+# Source files
+USER_SRCS += src/*.cc
+
+# User defines
+USER_DEFS = PROFILE=$(profile)
+USER_UNDEFS =
+USER_CPP_DEFS = FLUTTER_PLUGIN_IMPL CPP_PROFILE=$(profile)
```

Comment:
I was thinking that the profile information might be useful in building at some point - when the generator was creating a C++ code with all the known includes, some includes were not working depending on target profile/platform version. Some \#if could have solve that. But in the end, this plugin will not make use of that information. The generator now scans all the headers the tizen_interop is created from and extracts all the information we need.
As long as there is no conflict in callback name  - same name used for callbacks with incompatible signatures - we are ok.

Reply:
Thanks. If `PROFILE` and `CPP_PROFILE` are not likely to be used later, they might be removed.

Reply:
In fact, there's no need to add the `dart_api.c` file since `dart_api_dl.c` is implicitly compiled by the tool (and linked with plugins). Anyway, I'll take care of that part and maybe also push some other commits directly to the branch for general cleanup. Thanks for the great work!